lib/oli_web/live/delivery/manage_source_materials.ex

```diff
@@ -0,0 +1,251 @@
+defmodule OliWeb.Delivery.ManageSourceMaterials do
+  use Surface.LiveView, layout: {OliWeb.LayoutView, "live.html"}
+  use OliWeb.Common.Modal
+
+  alias Oli.Authoring.Course
+  alias Oli.Delivery.Sections
+  alias Oli.Delivery.Updates.Subscriber
+  alias Oli.Delivery.Updates.Worker
+  alias Oli.Publishing
+  alias Oli.Publishing.Publications.Publication
+  alias Oli.Publishing.Publications.PublicationDiff
+  alias OliWeb.Common.Breadcrumb
+  alias OliWeb.Delivery.ManageSourceMaterials.{ApplyUpdateModal, ProjectInfo, ProjectCard}
+  alias OliWeb.Router.Helpers, as: Routes
+  alias OliWeb.Sections.Mount
+
+  @title "Manage Source Materials"
+
+  data base_project_details, :struct
+  data current_publication, :map
+  data projects_remixed, :map
+  data section, :struct
+  data updates, :map
+  data updates_in_progress, :map
+
+  @spec set_breadcrumbs(atom | %{:slug => any, optional(any) => any}, any) :: [...]
+  def set_breadcrumbs(section, type) do
+    type
+    |> OliWeb.Sections.OverviewView.set_breadcrumbs(section)
+    |> breadcrumb(section)
+  end
+
+  def breadcrumb(previous, section) do
+    previous ++
+      [
+        Breadcrumb.new(%{
+          full_title: @title,
+          link: Routes.live_path(OliWeb.Endpoint, __MODULE__, section.slug)
+        })
+      ]
+  end
+
+  def mount(
+        _params,
+        %{"section_slug" => section_slug} = session,
+        socket
+      ) do
+    case Mount.for(section_slug, session) do
+      {:error, e} ->
+        Mount.handle_error(socket, {:error, e})
+
+      {user_type, _, section} ->
+        updates = Sections.check_for_available_publication_updates(section)
+        updates_in_progress = Sections.check_for_updates_in_progress(section)
+        base_project_details = Course.get_project!(section.base_project_id)
+
+        current_publication =
+          Sections.get_current_publication(section.id, base_project_details.id)
+
+        projects_remixed = Sections.get_projects_remixed(section.id, base_project_details.id)
+
+        Subscriber.subscribe_to_update_progress(section.id)
+
+        {:ok,
+         assign(socket,
+           title: @title,
+           section: section,
+           updates: updates,
+           updates_in_progress: updates_in_progress,
+           delivery_breadcrumb: true,
```

Comment:
Wdyt about instead define the `title` and this assign just as a surface data with a default?

Reply:
In fact I don't need it anymore because I only use it to define the title of the breadcrumb.